mkt/constants/applications.py

```diff
@@ -29,7 +29,15 @@ class DEVICE_GAIA(object):
     api_name = 'firefoxos'
 
 
-DEVICE_TYPE_LIST = [DEVICE_DESKTOP, DEVICE_MOBILE, DEVICE_TABLET, DEVICE_GAIA]
+class DEVICE_TV(object):
+    id = 5
+    name = _(u'Firefox OS TV')
+    class_name = 'firefoxos+tv'
+    api_name = 'firefoxos+tv'
```

Comment:
Also the "+" in api_name/class_name isn't coherent with Android which use "-", so that's wrong. Sorry about that, might have been a wrong suggestion from me (sadly both separators exist in fireplace, but shouldn't in zamboni)

Ideally we'd have:

```
class DESKTOP:
    'api_name': 'desktop'

class FIREFOXOS_MOBILE:
    'api_name': 'firefoxos_mobile'  # aliased to "firefoxos", but "firefoxos" should never be exposed as a choice

class FIREFOXOS_TV:
   'api_name': 'firefoxos_tv'

class ANDROID_MOBILE:
   'api_name': 'android_mobile'

class ANDROID_TABLET:
    'api_name': 'android_tablet'
```